events/src/main/kotlin/edu/artic/events/AllEventsAdapter.kt

```diff
@@ -12,10 +14,15 @@ import kotlinx.android.synthetic.main.cell_all_events_layout.view.*
  * @author Sameer Dhakal (Fuzz)
  */
 
-class AllEventsAdapter() : AutoHolderRecyclerViewAdapter<AllEventsCellViewModel>() {
+class AllEventsAdapter(val viewModel: AllEventsViewModel) : AutoHolderRecyclerViewAdapter<AllEventsCellViewModel>() {
 
     override fun View.onBindView(item: AllEventsCellViewModel, position: Int) {
 
+        clicks()
+                .defaultThrottle()
+                .subscribe { viewModel.onClickEvent(position, item.event) }
+                .disposedBy(item.viewDisposeBag)
```

Comment:
doing this will continue adding the event continuously forever as you scroll. not ideal.

Reply:
alright will update.